ReSharper.FSharp/src/FSharp.Psi.Daemon/src/Analyzers/LambdaAnalyzer.fs

```diff
@@ -84,45 +85,83 @@ type LambdaAnalyzer() =
 
         compareArgsRec expr 0 null
 
-    let isImplicitlyConvertedToDelegate (lambda: ILambdaExpr) =
+    let canBeNamedArg (expr: IBinaryAppExpr) =
+        if isNull expr.Operator then false else
+
+        let exprReference = expr.Operator.Reference
+        if exprReference.GetName() <> "=" then false else
+
+        let declaredElement = exprReference.Resolve().DeclaredElement.As<IFunction>()
+        isNull declaredElement
```

Comment:
IIRC `=` is never being resolved when expression is a named arg from the syntax point of view. Here're some examples to consider:

```fsharp
type T =
    static member M(a) = a + 1

T.M(a = 1)

let (=) a b = 1
T.M(a = 1)

let b = 1
T.M(b = 1)
```

<img width="269" alt="Screenshot 2022-08-19 at 18 39 00" src="https://user-images.githubusercontent.com/3923587/185655917-f1724df5-1c60-41f3-a269-4cff73557705.png">

Reply:
🤔